File: week_02/src/main/java/Question18.java
/*
Autor  = Zaid Nayeb
Date   = 18/11/2025

18.	Multiplication table of a number
Description: Print the multiplication table for a given number (up to 10 or N).
Example:
Input:
5
Output:
5 × 1 = 5
5 × 2 = 10
…
5 × 10 = 50

*/
public class Question18 {
    public static void main(String[] args) {
        multiTable(5,10);
    }
    public static void multiTable(int n, int m){
        for (int i = 1; i <= m; i++){
            System.out.println(n + " * " + i + " = "+ (i*n));
        }
    }
}

File: week_03/src/Question_18.js
// Author : Zaid Nayeb
// Date   : 2025/11/26
/*
18.	Pattern Printing – Right-aligned Triangle Example: 
Input: 
4 
Output: 
   *   
  **   
 ***   
**** 

*/

rightTriangle(10)
function rightTriangle(lines) {
    for (let i = 0; i < lines; i++) {
        for (let j = lines; j >= i; j--) {
            process.stdout.write("  ");
        }
        for (let k = 0; k <= i; k++) {
            process.stdout.write(" *");

        }
        console.log();
    }
}